app/routers/dogfinder.py

```diff
@@ -30,6 +30,9 @@
 UUID_FIELD = "uuid5"
 DOG_ID_FIELD = "dogId"
 
+# CHANGE THIS TO FALSE ON PRODUCTION
+IS_VERIFIED_FIELD_DEFAULT_VALUE = True
```

Comment:
shouldn't this be false? isn't that the point of the verify api?

Reply:
You didn't read the comment above it

Reply:
I set it to True so it will be easier to work with the search until we will have a UI for this